server/queue/queue-rabbitmq/src/test/java/org/apache/james/queue/rabbitmq/RabbitMQMailQueueTest.java

```diff
@@ -216,13 +219,17 @@ void mailQueueShouldBeInitializedWhenCreating(CassandraCluster cassandra) {
     @Override
     public void constructorShouldRegisterGetQueueSizeGauge(MailQueueMetricExtension.MailQueueMetricTestSystem testSystem) {
     }
+
+    private Function<Integer, String> namePatternForSlice(int sliceId) {
+        return i -> sliceId + "-" + i;
+    }
     
-    private void enqueueMailsInSlice(int slice, int emailCount) {
+    private void enqueueSomeMails(Function<Integer, String> namePattern, int emailCount) {
```

Comment:
ok, you fixed things here but because of code duplication it's not fixed in previous file.